TOC-tidb-cloud.md

```diff
@@ -237,7 +237,6 @@
     - [Migrate and Merge MySQL Shards of Large Datasets](/tidb-cloud/migrate-sql-shards.md)
     - [Migrate from TiDB Self-Managed to TiDB Cloud](/tidb-cloud/migrate-from-op-tidb.md)
     - [Migrate from MySQL-Compatible Databases Using AWS DMS](/tidb-cloud/migrate-from-mysql-using-aws-dms.md)
-    - [Migrate from Amazon RDS for Oracle Using AWS DMS](/tidb-cloud/migrate-from-oracle-using-aws-dms.md)
```

Comment:
reason: This first paragraph in this doc mentions that "This document describes a step-by-step example of how to migrate data from Amazon RDS for Oracle to [TiDB Cloud Starter](https://tidbcloud.com/clusters/create-cluster) using AWS Database Migration Service (AWS DMS)."